client/src/main/java/org/apache/uniffle/client/util/RssClientConfig.java

```diff
@@ -65,4 +65,8 @@ public class RssClientConfig {
   public static final int RSS_ACCESS_TIMEOUT_MS_DEFAULT_VALUE = 10000;
   public static final String RSS_DYNAMIC_CLIENT_CONF_ENABLED = "rss.dynamicClientConf.enabled";
   public static final boolean RSS_DYNAMIC_CLIENT_CONF_ENABLED_DEFAULT_VALUE = true;
+
+  public static final String RSS_CLIENT_ASSIGNMENT_SHUFFLE_SERVER_NUMBER =
+      "rss.client.assignment.shuffle.server.number";
```

Comment:
`rss.client.assignment.shuffle.node.max`
Could we use  a similar style name with the coordinator's configuration?

Reply:
OK